### ISSUE TITLE
Editing comment in comment tab creates new post draft

### ISSUE THREAD

**Issue by espoem** [busyorg/busy#963]
#### Expected behavior

Editing a comment in the comment tab in user's profile should open the comment for editing.

#### Actual behavior

Editing a comment in comment tab opens a new post draft with the content from the comment.

![comments-edit](https://user-images.githubusercontent.com/16621517/32240199-fbf55c84-be6c-11e7-9805-56faff3c61ee.gif)

#### How to reproduce

1. Go to your profile
2. See comments
3. Edit your comment

* Browser: Chrome
* Operating system: Windows 10
* Credit: `/@eqko/busy-org-beta-bughunting-edit-comment-on-commentpage-makes-new-post-draft-instead-of-editing-the-comment`

**Comment by Kennybll** [comment]
@bonustrack, I see that this feed uses StoryPreview. If we allow an edit to this we would have to make another page to edit comments. Unless we use EmbeddedCommentForm, the only thing is we would have to edit the whole comment not the preview. It's fine to do that, I'm just not sure we should do that. It would be weird to see a preview of a long comment then see the whole thing to edit.

But if we wan't to use EmbeddedCommentForm I started to work on that. Here is what I think it should look like:
![image](https://user-images.githubusercontent.com/14302151/33858200-cd01d308-de9c-11e7-8497-f239f48929dc.png)

I started to work on it, but I thought to ask how we should handle this first.
